scripts/trivia.js

```diff
@@ -2158,6 +2191,10 @@ addAdminCommand(["search"], function (src, commandData, channel) {
 }, "Allows you to search through the questions, format /search [query]. Only matches whole words.");
 
 addAdminCommand(["apropos"], function (src, commandData, channel) {
+    if (trivData.eventFlag) {
+        Trivia.sendPM(src, "You cannot use /apropos during event games!", channel);
```

Comment:
As something that pretty much gives you the answers instantly, there should at least be some broadcast in VR that someone used apropos on what question, I feel :x